src/pudl/metadata/resources/eia861.py

```diff
@@ -642,6 +786,7 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_utility_data_rto": {
+        "description": "The RTOs that a utility operates in.",
```

Comment:
Is it really just RTOs? Or does EIA-861 just default to using RTO and not mention ISO?  I guess I'm still not 100% sure what the difference is between an ISO and an RTO if there is one.

Reply:
I think they are effectively the same thing, RTOs are just bigger.